index.html

```diff
@@ -5,104 +5,123 @@
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    <div class="boxoneone">
-        <img class="image"src="image_big1_3.jpg" alt="3">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    <div class="boxoneone">
-        <img class="image"src="image_big2_1.jpg" alt="image1">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    <div class="boxoneone">
-        <img class="image"src="image_big2_2.jpg" alt="image2">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    <div class="boxoneone">
-        <img class="image"src="image_big2_3.jpg" alt="3">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    <div class="boxoneone">
-        <img class="image"src="image_big3_1.jpg" alt="image1">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    <div class="boxoneone">
-        <img class="image"src="image_big3_2.jpg" alt="image2">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>                 
-    <div class="boxoneone">
-        <img class="image"src="image_big3_3.jpg" alt="3">
-        <h2 class="topleft1">i'am hangry desinger</h2>
-        <p class="bottonleft1">TheCodrops always hungry to lern.</p>
-    </div>
-    </div>
+    <main>
+        <h2 class="color2 text2">recent works</h2>
+        <p>Lorem ipsum dolor sit amet, pri ea cotidieque vituperatoribus, scaevola convenire maluisset at vel. 
+                     Et nulla referrentur instructior eum.</p>
+    </main>
 
-    <div class="work">
-            <img class="obraz" src="tlo2.jpg" alt="work"/>
-            <h1>How we work</h1>
-            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio enim modi sapiente quisquam non aliquid, provident sunt qui reprehenderit! Tempore inventore consequatur nesciunt nostrum praesentium sunt illum architecto atque, suscipit soluta fuga, amet sed dolorem nobis neque dolores. Ipsa ullam accusantium harum veniam cupiditate magni officiis, dolorum sint atque? Veniam repellendus amet itaque, magnam. Nulla sapiente nostrum delectus consequuntur sunt quos. Veritatis enim provident, iure odio in molestias dolor. Repudiandae quas, temporibus voluptates aperiam vel nulla nobis minima cumque aliquid!</p>
-    </div>
+    <section class="card-section">
+        <div class="card">
+            <h5 class="color1 text2">i'm hangry desinger</h5>
+            <p class="color1">TheCodrops always hungry to lern.</p>
```

Comment:
Ok, oba powyższe elementy pozycjonujemy niezależnie. Mowa o `h5` i `p`. Obu nadajemy `position: absolute` i różne wartości dla `left`, `bottom` itp. Po pierwsze, możemy skrócić ten kod, a po drugie, ponieważ te elementy są ze sobą w pewien sposób powiązane, powinniśmy je złączyć w jedną grupę. Mam na myśli opakować je w jakieś opakowanie i pozycjonować tylko to opakowanie. Dzięki temu unikniemy problemu, że na telefonie czy tablecie te dwa elementy się jakoś rozjeżdżają.